shell/src/main/java/alluxio/cli/fs/command/FreeWorkerCommand.java

```diff
@@ -82,9 +83,13 @@ public int run(CommandLine cl) throws AlluxioException, IOException {
     }
 
     // 3. Remove target worker metadata.
+    // TODO(jiacheng): further consolidate how to use this FreeWorker command in the procedure
     try (CloseableResource<BlockMasterClient> blockMasterClient =
                  mFsContext.acquireBlockMasterClientResource()) {
-      blockMasterClient.get().removeDecommissionedWorker(workerName);
+      RemoveDecommissionedWorkerPOptions options = RemoveDecommissionedWorkerPOptions.newBuilder()
```

Comment:
@dbw9580 do you know what this intends to do? I actually don't understand why this calls RemoveDecommissionedWorker

Reply:
I think this is to remove any related metadata (blocks held by this worker, mostly) of this worker in the block master, so the memory can be released.

Reply:
ok so this logic here is not really a correct use of DecommissionWorker. I will remove this call as a whole from `FreeWorkerCommand`